Bottom: 7d9e5cc230e2810cf4b2e08f71e5100c4d4d5941
Top:    739e2ac1d4183624ea78971f116d0b6be0266833
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-11 10:41:02 -0700

rstream: Clarify use of async test option

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/rstream.c b/examples/rstream.c
index a214b1a..b73d59a 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -554,7 +554,7 @@ int main(int argc, char **argv)
 			printf("\t[-p port_number]\n");
 			printf("\t[-T test_option]\n");
 			printf("\t    s|sockets - use standard tcp/ip sockets\n");
-			printf("\t    a|async - asynchronous operation\n");
+			printf("\t    a|async - asynchronous operation (use poll)\n");
 			printf("\t    b|blocking - use blocking calls\n");
 			printf("\t    n|nonblocking - use nonblocking calls\n");
 			printf("\t    v|verify - verify data\n");
